src/elements/tabs/tabs.tsx

```diff
@@ -94,7 +94,7 @@ export const Tabs = ({
   )?.actions
   return (
     <TabsRoot value={value} onValueChange={onValueChange} {...props}>
-      <div className="flex justify-between">
+      <div className="flex justify-between sticky top-14 z-30 bg-background">
```

Comment:
I used top-14 and z-30 because of the following `<header className="sticky top-0 left-0 z-20 w-full border-b select-none shadow transition-colors bg-background">`.
Is it a bad way?

Reply:
I mean If we are using `top-0`, it hides the `<header...>`. Is it okay?

Reply:
No, it's not okay. That's your task. Make it work using z-30. Use z-10, header and tabs always should be visible

Reply:
Uhm? Z-10 ? Or z 30 ?